azurelinuxagent/ga/update.py

```diff
@@ -815,17 +853,6 @@ def _is_orphaned(self):
 
         return fileutil.read_file(conf.get_agent_pid_file_path()) != ustr(parent_pid)
 
-    def _is_version_eligible(self, version):
```

Comment:
This function was only being used in 1 place (during auto-update) and due to the condition below, it was always True (`if version == CURRENT_VERSION and is_current_agent_installed(): return True`). Since it was not adding much value to the code, removed it